src/main/de/linearbits/preferences/EditorBoolean.java

```diff
@@ -83,7 +95,12 @@ Boolean parse(final String s) {
     void setValue(Object t) {
         setInitialValue((Boolean) t);
         checkbox.setSelection((Boolean) t);
-        checkbox.setText((Boolean) t ? getDialog().getConfiguration().getStringYes() : getDialog().getConfiguration().getStringNo());
+        if (!hideLabel) {
+            checkbox.setText((Boolean) t ? getDialog().getConfiguration().getStringYes() : getDialog().getConfiguration().getStringNo());
+        } else {
+            checkbox.setText("");
+        }
+        
```

Comment:
This blank line seems to be redundant as well (but this is just personal preference; no need to change if you like it this way).